src/App.js

```diff
@@ -1,16 +1,98 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+
+if(localStorage.getItem('myData')===null){
+  var TodoData =[];
+} else {
+  TodoData = JSON.parse(localStorage.getItem('myData'));
+}
+
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      todos: TodoData,
+      inputText:""
+    };
+  }
+
+  addTodo = event => {
+    event.preventDefault();
+    this.setState({
+      todos:[
+        ...this.state.todos,{task:this.state.inputText,id:Date.now(),
```

Comment:
This is good.  Looking forward you need to be careful about using Dante.now() as the id.  If your application creates a bunch of things at once you could end up with non-unique ids.  In this case it works perfectly but in other cases you just want to be aware of its limitations.